scripts/dts/python-devicetree/src/devicetree/dtlib.py

```diff
@@ -602,11 +628,11 @@ def __str__(self):
                 # end - 1 to strip off the null terminator
                 s += f' "{_decode_and_escape(self.value[pos:end - 1])}"'
                 if end != len(self.value):
-                    s += ","
+                    s += "," + newline
```

Comment:
```suggestion
                    s += f",{newline}"
```

They are the same, a suggestion :)